codalab/model/bundle_model.py

```diff
@@ -1210,7 +1210,7 @@ def get_bundle_states(self, uuids):
     def get_bundle_storage_info(self, uuid):
         result_dict = self.get_bundle_storage_infos([uuid])
         if uuid not in result_dict:
-            raise NotFoundError('Could not find bundle with uuid %s' % uuid)
```

Comment:
Should at least log something?

Reply:
No, because this would happen pretty often. Any bundle which was uploaded before the introduction of Blob Storage will trigger this code path, because `is_dir` and `storage_type` wouldn't be set.